projects/1.20/assets/the-dawn-era/dawnera/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+  "item.dawnera.alligator_gar": "鳄雀鳝",
+  "item.dawnera.alligator_gar_spawn_egg": "鳄雀鳝刷怪蛋",
+  "item.dawnera.bone_whistle": "骨哨",
+  "item.dawnera.bowl_of_caviar": "鲟鱼鱼子酱",
+  "item.dawnera.bowl_of_sabertooth_salmon_roe": "剑齿鲑鱼子酱",
+  "item.dawnera.bowl_of_sabertooth_salmon_soup": "剑齿鲑汤",
+  "item.dawnera.caviar_sandwich": "鱼子三明治",
+  "item.dawnera.ceratosaurus_armor": "角鼻龙铠",
+  "item.dawnera.ceratosaurus_saddle": "角鼻龙鞍",
+  "item.dawnera.ceratosaurus_spawn_egg": "角鼻龙刷怪蛋",
+  "item.dawnera.coelacanth": "空棘鱼",
+  "item.dawnera.coelacanth_spawn_egg": "空棘鱼刷怪蛋",
+  "item.dawnera.cooked_alligator_gar": "熟鳄雀鳝",
+  "item.dawnera.cooked_coelacanth": "熟空棘鱼",
+  "item.dawnera.cooked_psittacosaurus_meat": "熟鹦鹉嘴龙肉",
+  "item.dawnera.cooked_sabertooth_salmon": "熟剑齿鲑",
+  "item.dawnera.cooked_struthiomimus_meat": "熟似鸵龙肉",
+  "item.dawnera.cooked_sturgeon": "熟鲟鱼",
+  "item.dawnera.diabloceratops_armor": "魔鬼角龙铠",
+  "item.dawnera.diabloceratops_saddle": "魔鬼角龙鞍",
+  "item.dawnera.diabloceratops_spawn_egg": "魔鬼角龙刷怪蛋",
+  "item.dawnera.dimorphodon_spawn_egg": "双形齿龙刷怪蛋",
+  "item.dawnera.dunkleosteus_spawn_egg": "邓氏鱼刷怪蛋",
+  "item.dawnera.inspecting_spyglass": "检查望远镜",
+  "item.dawnera.mixosaurus_spawn_egg": "混鱼龙刷怪蛋",
+  "item.dawnera.pond_snail_spawn_egg": "椎实螺刷怪蛋",
+  "item.dawnera.psittaco": "鹦鹉嘴龙塔可",
+  "item.dawnera.psittacosaurus_meat": "鹦鹉嘴龙肉",
+  "item.dawnera.psittacosaurus_spawn_egg": "鹦鹉嘴龙刷怪蛋",
+  "item.dawnera.sabertooth_salmon": "剑齿鲑",
+  "item.dawnera.sabertooth_salmon_spawn_egg": "剑齿鲑刷怪蛋",
+  "item.dawnera.struthiomimus_meat": "似鸵龙肉",
+  "item.dawnera.struthiomimus_spawn_egg": "似鸵龙刷怪蛋",
+  "item.dawnera.sturgeon": "鲟鱼",
+  "item.dawnera.sturgeon_spawn_egg": "鲟鱼刷怪蛋",
+  "item.dawnera.triceratops_armor": "三角龙铠",
+  "item.dawnera.triceratops_horn": "三角龙角",
+  "item.dawnera.triceratops_saddle": "三角龙鞍",
+  "item.dawnera.triceratops_spawn_egg": "三角龙刷怪蛋",
+  "item.dawnera.tyrannosaurus_armor": "霸王龙铠",
+  "item.dawnera.tyrannosaurus_saddle": "霸王龙鞍",
+  "item.dawnera.tyrannosaurus_spawn_egg": "霸王龙刷怪蛋",
+  "item.dawnera.worm": "蚯蚓",
+  "item.dawnera.worm_spawn_egg": "蚯蚓刷怪蛋",
+  "key.categories.dawnera": "破晓纪元",
+  "key.dawnera.attack": "attack",
+  "key.dawnera.open_menu": "打开交互菜单",
+  "key.dawnera.roar": "吼叫",
+  "movement.dawnera.follow": "follow",
+  "movement.dawnera.wait": "wait",
+  "movement.dawnera.wander": "wander",
+  "msg.dawnera.behaviour.aggressive": "%1$s现在会攻击目标。",
+  "msg.dawnera.behaviour.ignore": "%1$s现在会无视目标。",
+  "msg.dawnera.behaviour.passive": "%1$s现在会反击目标。",
+  "msg.dawnera.order.allow_eat": "%1$s现在被允许进食。",
+  "msg.dawnera.order.deny_eat": "%1$s现在被禁止进食。",
+  "msg.dawnera.order.follow": "%1$s现在会跟随你。",
+  "msg.dawnera.order.gathering": "%1$s现在会采集资源。",
+  "msg.dawnera.order.no_order": "%1$s现在为空闲状态。",
+  "msg.dawnera.order.private": "%1$s现在只能被你骑乘。",
+  "msg.dawnera.order.public": "%1$s现在可以被任何人骑乘。",
+  "msg.dawnera.order.wait": "%1$s现在会原地等待。",
+  "msg.dawnera.order.wander": "%1$s现在会在周围徘徊。",
+  "msg.dawnera.tame": "成功驯服了%1$s!",
```

```suggestion
  "msg.dawnera.tame": "成功驯服了%1$s！",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。